lib/KCard/index.vue

```diff
@@ -0,0 +1,226 @@
+<template>
+
+  <BaseCard
+    :title="title"
+    :headingLevel="headingLevel"
+    :titleLines="titleLines"
+    :to="to"
+  >
+    <template #default>
+      <aside
+        v-if="layout === 'vertical' && thumbnailDisplay !== 'none'"
+      >
+        <KImg
+          :src="thumbnailSrc"
+          :isDecorative="true"
```

Comment:
I think not all images on a card are always decorative.  From my understanding the `:isDecorative `property used to indicate to assistive technologies (screen readers) whether the image provides essential information or not. Screen readers might skip decorative images to improve reading efficiency for visually impaired users.

Your intuition is right. It's very common to have a descriptive text alternative (alt text) for informative thumbnails. Including a `thumbnailAlt` property would allow us to provide this description, which would be beneficial for visually impaired users as well. https://www.w3.org/WAI/tutorials/images/decorative/